.markdownlint-cli2.yaml

```diff
@@ -2,3 +2,5 @@ config:
   line-length: false
   no-emphasis-as-header: false
   first-line-heading: false
+ignores:
+  - pkg
```

Comment:
Going to fix this after importing?

Reply:
Maybe. Added more detail to the commit message.
Some of the packages that we have are forks of older, unmaintained packages.
Not sure if we should edit the README's there or not.